tests/SimplifyTest.elm

```diff
@@ -5887,7 +5887,7 @@ a = String.repeat 0 str
                     |> Review.Test.run ruleWithDefaults
                     |> Review.Test.expectErrors
                         [ Review.Test.error
-                            { message = "Using String.repeat with length 0 will result in \"\""
+                            { message = "Using String.repeat with length 0 will always result in \"\""
```

Comment:
Not sure the "always" in the `message` is relevant, it feels a bit unnecessary.

Reply:
Is removed in a later commit